package.json

```diff
@@ -41,7 +41,7 @@
     "ThirdPartyNotices.txt"
   ],
   "browser": {
-    "./es/lib/policies/msRestUserAgentPolicy.js": "./es/lib/policies/msRestUserAgentPolicy.stub.js",
+    "./es/lib/policies/telemetry/msRestUserAgentPolicy.js": "./es/lib/policies/telemetry/msRestUserAgentPolicy.browser.js",
```

Comment:
@kpajdzik - Was there a reason to add telemetry in the file path over here? It doesn't match the relative locations of those files. There is no telemetry folder inside `./lib/policies/` ?

Reference https://github.com/Azure/ms-rest-js/issues/281

Reply:
I believe that's a result of moving the files into and out of the "telemetry" directory as a result of code review.

Reply:
I see. I think this is the root cause of the import os problem. Can you please fix this and publish a new version to latest tag?